src/OpenTK/Math/Vector3.cs

```diff
@@ -257,6 +257,36 @@ public void NormalizeFast()
         public static readonly int SizeInBytes = Marshal.SizeOf(new Vector3());
 
         /// <summary>
+        /// Defines a unit-length Vector3 that points upward.
```

Comment:
OpenTK should not care what handness the coordinate system used is. Especially with Vulkan coming which is right-handed by default vs OpenGL left handed by default.